xml/System.Numerics/Vector`1.xml

```diff
@@ -214,13 +247,15 @@
         <Parameter Name="values" Type="T[]" />
       </Parameters>
       <Docs>
-        <param name="values">A numeric array.</param>
+        <param name="values">The values to add to the vector, as an array of objects of type T. The array must contain at least <see cref="P:System.Numerics.Vector`1.Count" /> elements.</param>
```

Comment:
Maybe the following, to make it clear in IntelliSense:
```suggestion
The values to add to the vector, as an array of objects of type T. The array must contain at least <see cref="P:System.Numerics.Vector`1.Count" /> elements and only the first <see cref="P:System.Numerics.Vector`1.Count" /> elements are used.
```

Reply:
Does the change made in this repo propagate to the XML docs shown in the VS? I genuinely do not know if it does; thought it was only the XML comments that make into it.

Reply:
I believe that the intellisense for the .net libraries are specifically built from these xml files, not the source code /// comments.

Reply:
It's a mix. For .NET Core and .NET Standard APIs, the IntelliSense is generated from this repo. For Platform extensions, no.

But only future versions of .NET Core and .NET Standard IntelliSense will get the improvement.